### ISSUE TITLE
Possible capability issue with Docker CE on OpenWrt

### ISSUE THREAD

**Issue by hsghost** [v2rayA/v2rayA#435]
### Version, approach of installation, operating system

1. What version of v2rayA are you using?

> 1.5.6.2

2. What approach of installation do you install V2rayA?

> Docker or opkg

3. What is your operating system?

> OpenWrt Snapshot 2022-02-27

### Description
v2rayA seems to having some captability issue with the _Docker Community Edition_ on a recent build of OpenWrt snapshot. Especially, once the `dockerd` package is installed through `opkg`, a previously configured and well running v2rayA instance would get affected, making the router stops working. An effort to move the service into a docker container didn't solve the problem, either. Both configurations share the same outcome: when the `xray-core` is down, the router is able to ping public IPs, able to access the Internet, and so are the client computers; once the `xray-core` is up, however, only the router itself have access to the Internet, the clients lost connectivity completely, whether it is a `PROXY` or `DIRECT` destination in the rules of v2rayA. Consulting the logs reveals that there's no client-related connections going through v2rayA at all!

A simple delve shows the potential culprit being under `iptables`. Especially, the `tproxy` mechanism used by both packages seemes to mess up. Changing the method to `redirect` showed a little light, as I could see entries from the clients comming into the log. Yet, the webpages still couldn’t be loaded successfully on client machines, so problem persists. And as it’s a router use case, I definitely would like `tproxy` to handle all the UDP traffic for the clients, instead of `redirect`.

However, my knowledge to `iptables` is sitll premature, so the following is what I can get thus far, without knowing what to change, or is it possible to change to even make it to work: 

```
# iptables -t nat -nvL
Chain PREROUTING (policy ACCEPT 4822 packets, 2061K bytes)
 pkts bytes target     prot opt in     out     source               destination         
  256 16926 DOCKER     all  --  *      *       0.0.0.0/0            0.0.0.0/0            ADDRTYPE match dst-type LOCAL

Chain INPUT (policy ACCEPT 349 packets, 22758 bytes)
 pkts bytes target     prot opt in     out     source               destination         

Chain OUTPUT (policy ACCEPT 457 packets, 30943 bytes)
 pkts bytes target     prot opt in     out     source               destination         
    0     0 DOCKER     all  --  *      *       0.0.0.0/0           !127.0.0.0/8          ADDRTYPE match dst-type LOCAL
    0     0 DOCKER     all  --  *      *       0.0.0.0/0           !127.0.0.0/8          ADDRTYPE match dst-type LOCAL

Chain POSTROUTING (policy ACCEPT 226 packets, 15768 bytes)
 pkts bytes target     prot opt in     out     source               destination         
    0     0 MASQUERADE  all  --  *      !docker0  172.17.0.0/16        0.0.0.0/0           

Chain DOCKER (3 references)
 pkts bytes target     prot opt in     out     source               destination         
    0     0 RETURN     all  --  docker0 *       0.0.0.0/0            0.0.0.0/0
```

As you can see there's no rule related to v2rayA, and when `dockerd` gets removed the above command only shows empty tables. I didn't check before the initial installation of `dockerd`, so whether it was being wiped by it, or it was simply because the core I chose was xray, I have no idea.

I also tried, an uninstallation and reinstallation of v2rayA did nothing to remedy. Fortunately, uninstalling the `dockerd` package restores my connection. Yet, on my _GL.iNet GL-B1300_, with a typical load average under 0.5, I definitely would like to feed its quad core CPU something extra to crunch with.

So please help, thank you!

**Comment by yuyue9284** [comment]
Try `sysctl net.bridge.bridge-nf-call-iptables=0`

**Comment by 1715173329** [comment]
I don't think this is something related to v2rayA. Instead, it's caused by the OpenWrt itself.
